### PR TITLE
Dialog metrics

### PR DESCRIPTION
**Pull request by vincenthoogsteder**
not for merge, only review

### CODE REVIEW

File: notebooks/dialog-metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f1bda0b9-bfad-4466-a887-a5b069d9d3bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# first, add the first session to the earlier created interactive_events\n",
```
Comment (hendrik-obj) [inline notebooks/dialog-metrics.ipynb:450]:
```suggestion
    "# first, add a column labeling a session as first session to the earlier created interactive_events\n",
```

File: notebooks/dialog-metrics.ipynb [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "129ae897-e2c8-4af3-9612-40812a60810a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add first session to the data, also non-interactive events, to define the first session duration\n",
```
Comment (hendrik-obj) [inline notebooks/dialog-metrics.ipynb:484]:
```suggestion
    "# add a column labeling a session as first session to the data (including non-interactive events) to define the first session duration\n",
```

**Review by hendrik-obj — COMMENTED** [review]
all works, some small comments on stuff the _could_ be done differently.

File: notebooks/dialog-metrics.ipynb [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2d398967-3d01-4c73-89e0-af6a70f957af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# instantiate the model hub\n",
+    "modelhub = ModelHub(time_aggregation='YYYY-MM-DD')\n",
```
[the remark this answers is not in the document]
Comment (vincenthoogsteder) [reply notebooks/dialog-metrics.ipynb:57]: good point, will add

**Comment by ivarpruijn** [comment]
recheck

**Pull request closed without merging**